src/main.c

```diff
@@ -296,8 +297,10 @@ main (int argc, char *argv[])
 
           if (rename_status == 0)
           {
-            printf ("'%s' -> '%s'\n", file.main_argv, file.dest_name);
-
+			if (verbose)  
```

Comment:
There are still some tabs preceding the if statement here, and trailing white space (see comment at L52)